### PR TITLE
Exclude /dev/* from images

### LINKED ISSUE #356: new multiarch build includes /dev/null in image
Apparently recent change introduced `/dev/null` in the image. This has not really any effect when running the image, but it broke scripts that does things like:

````
mkdir tmp && docker export $(docker create alpine:3.6) | tar -x -C tmp
````

I think we should remove the `/dev/*` since docker manages that dynamically and to preserve backwards compatibility.

### PR DESCRIPTION
**Pull request by ncopa**
fixes #356

### CODE REVIEW

**Comment by tianon** [comment]
Is this going to also repack the release tarballs that we're using for multiarch `alpine:*`? (I thought we were embedding the tarballs from alpinelinux.org directly as-is?)

**Comment by andyshinn** [comment]
It will not, it looks like builder only changes. Does that need to happen as well?

**Comment by andyshinn** [comment]
I guess this is an argument for moving the download vs build switch logic to the builder instead. That way, that could both reuse this tar step.

File: builder/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.2
-COPY scripts/mkimage-alpine.bash scripts/apk-install /
```
Comment (andyshinn) [inline builder/Dockerfile]: The `scripts/apk-install` file is still used by some `gliderlabs` org images so we may want to keep it (the failed builds are for this).

**Pull request merged**